docker-plugin/src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/config.jelly

```diff
@@ -47,11 +47,10 @@
         </f:entry>
 
         <f:advanced>
-
             <!-- reworked strategy choosed from core -->
             <f:dropdownList name="retentionStrategy" title="${%Availability}"
                             help="/help/system-config/master-slave/availability.html">
-                <j:forEach var="d" items="${h.getRetentionStrategyDescriptors()}">
+                <j:forEach var="d" items="${instance.descriptor.dockerRetentionStrategies}">
```

Comment:
Sounds like something else is broken then. In normal form construction `descriptor` is always set.